test/cases/compile_errors/unhandled_enum_value_in_switch_with_enum_declared_in_other_file.zig

```diff
@@ -0,0 +1,16 @@
+const std = @import("std");
+
+pub export fn entry1() void {
+    const order: std.math.Order = .lt;
+    switch (order) {}
+}
+
+// error
+// backend=stage2
+// target=native
+//
+// :5:5: error: switch must handle all possibilities
+// :1363:5: note: unhandled enumeration value: 'lt'
+// :1366:5: note: unhandled enumeration value: 'eq'
+// :1369:5: note: unhandled enumeration value: 'gt'
+// :1361:19: note: enum 'math.Order' declared here
```

Comment:
Use `:?:?:` so that this test doesn't break every time `math.zig` is slightly altered.